src/SAMLTrace.js

```diff
@@ -630,21 +634,23 @@ SAMLTrace.TraceWindow.prototype = {
     case 'application/font-woff2':
     case 'application/x-font-ttf':
     case 'application/x-font-woff':
+    case 'font/woff2':
```

Comment:
This one should already be filtered by the `case` on line 612/616

Reply:
You're absolutely right. Can't tell why I thought, this'd be required. I'll revert it.